src/service/provider/provider.service.ts

```diff
@@ -59,8 +70,8 @@ export class ProviderService {
         },
       ]
     >`select count(distinct provider)::int,
-             100 * avg(success_rate) as "averageSuccessRate"
-      from provider_retrievability_daily;`;
+             100 * avg(avg_retrievability_success_rate) as "averageSuccessRate"
+      from providers_weekly where week = ${lastWeek.week};`;
```

Comment:
I know that `week` is a date type field and there's no way it could contain something malicious, but can we make it SQL-Injection safe just to be extra sure and to give good example to junior devs?

Reply:
We are safe by using $queryRaw. Only if we used $queryRawUnsafe that could be risky.

Reply:
Huh, TIL, I had no idea about [tagged templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates) in Javascript. I was sure it first gets interpolated and then passed to the function. Thanks!